scripts/upgrade_chromedriver.js

```diff
@@ -31,7 +31,7 @@ switch (process.platform) {
 
   case 'darwin':
     versionCheckCommands.push(
-      '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome --version'
+      '/Applications/Google\\ Chrome.app/Contents/MacOS/Google\\ Chrome --version'
```

Comment:
Any other strings we need to check for path escaping?

Reply:
nah just this one.